### ISSUE TITLE
Why not contribute / port to official Semantic Org Angular project?

### ISSUE THREAD

**Issue by lukeclifton** [angularify/angular-semantic-ui#56]
Hey.

I see that Semantic Org are working on official project for Angular support.
https://github.com/Semantic-Org/Semantic-UI-Angular

Did this repo come first or something? I would like to contribute but seems to make sense to contribute to official project. Although looks like more progress has been made here.

Is there any reason for the two separate efforts? Could they be merged?

**Comment by dv336699** [comment]
Yep, this project was released two years ago while the official was one year ago. Everyone should be using the official one and send PR there instead.

**Comment by rbecheras** [comment]
@0xAX : What is your thoughts about this ?
